kubernetes-deployment/bootstrap.py

```diff
@@ -28,6 +28,7 @@
 import time
 import logging
 import logging.config
+from clusterObjectModel import objectModelFactory
```

Comment:
I suggest we follow the coding-style practices of PEP8: https://www.python.org/dev/peps/pep-0008/